Common/cpp/AnimatedSensor/AnimatedSensorModule.cpp

```diff
@@ -35,37 +32,35 @@ jsi::Value AnimatedSensorModule::registerSensor(
       iosReferenceFrame.asNumber(),
       [sensorType,
        shareableHandler,
-       weakRuntimeHelper = std::weak_ptr<JSRuntimeHelper>(runtimeHelper)](
+       weakUiWorkletRuntime = std::weak_ptr<WorkletRuntime>(uiWorkletRuntime)](
           double newValues[], int orientationDegrees) {
-        auto runtimeHelper = weakRuntimeHelper.lock();
-        if (runtimeHelper == nullptr || runtimeHelper->uiRuntimeDestroyed) {
+        auto uiWorkletRuntime = weakUiWorkletRuntime.lock();
+        if (uiWorkletRuntime == nullptr) {
           return;
         }
 
-        auto &rt = *runtimeHelper->uiRuntime();
-        auto handler = shareableHandler->getJSValue(rt);
+        jsi::Runtime &uiRuntime = uiWorkletRuntime->getRuntime();
```

Comment:
Renamed `getRuntime` with `getJSIRuntime` in e9968126fc8c1abd581a1bd9c593db9c6c48446e.

There's also `getUIRuntime` method so that `NativeProxy` and `RNRuntimeDecorator` can access the UI runtime (as `jsi::Runtime &`), should we also rename it to `getUIJSIRuntime`?